overkilling/overkill-todo-monolith-api
# Add /todos endpoint returning a preset of todos

### Description

Closes #1 

Exposes a `/todos` endpoint, currently serving a hardcoded set of todos.

It does not represent the best way of serving an API or interacting with the DB. However, it is a basic [walking skeleton](https://wiki.c2.com/?WalkingSkeleton) of an API with just enough functionality to count as a backend app.

Future PRs will gradually improve the functionality, techniques, and style. Hopefully.

 ### How to test

Steps to test :
1. Startup API
2. Make a request to `/todos`
3. Verify a stub list of todos is return
